tests/cli/backup/test_backup_command.py

```diff
@@ -30,7 +30,7 @@ def test_backup_flow_cells(
     )
 
     # GIVEN an encrypted flow cell
-    flow_cells_dir = Path(cg_context.backup.encryption_directories.current, flow_cell_full_name)
+    flow_cells_dir = Path(cg_context.encryption.encryption_dir, flow_cell_full_name)
```

Comment:
```suggestion
    flow_cells_encrypt_dir = Path(cg_context.encryption.encryption_dir, flow_cell_full_name)
```
flow_cells_dir == /home/proj/production/flow_cells   - might be confusing at some point, but can also be argued that it is isolated within the function

Reply:
Fixed